debian/open62541-dev.install

```diff
@@ -1,5 +1,2 @@
-usr/include/*
-usr/lib/lib{*.a,*.so,*.so.*}
-usr/lib/cmake/open62541/*
```

Comment:
I guess we need this line since we have some custom CMake macros which are added to the dev package.

You can check the content here:

https://launchpad.net/~open62541-team/+archive/ubuntu/daily/+files/open62541-dev_0.4-0~201901132115~ubuntu19.04.1_arm64.deb

Reply:
hmm, not sure that I've got it...
For the open62541-dev package is a dependency in debian/control to libopen62541 and the dependency ensures that the lib will be installed if installation of open62541-dev is requested. Therefore everything in usr/lib (even usr/lib/cmake/open62541/*) will be installed if one installs open62541-dev.